### PR TITLE
Run security tests in separate step

### LINKED ISSUE #644: Consider using dedicated container for Glue
Instead of installing ruby and glue twice - see #633

### LINKED ISSUE #645: Run security tests as separate check
see #633

### PR DESCRIPTION
**Pull request by nataly87s**
closes #645 
closes #644

### CODE REVIEW

**Comment by Yshayy** [comment]
Looks really cool

File: deployments/dev/zap/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+FROM owasp/zap2docker-bare
+
+RUN zap.sh -cmd -addonupdate -addoninstall pscanrulesAlpha -addoninstall pscanrulesBeta
```
Comment (omerlh) [inline deployments/dev/zap/Dockerfile]: Nice!

**Comment by omerlh** [comment]
Awesome! 
One thing missing - before shutting down Zap, you need to make sure the passive scan ended. Glue is doing that before reading the alerts. By not doing that, the results are unexpected - kind of race condition.
Also, worth checking with Zap's API that it did scan our URLs, to make sure there were no bugs.

**Comment by nataly87s** [comment]
How do I check that?

**Comment by omerlh** [comment]
See OWASP/glue#93 - it's in ruby but should be simple to understand the API request.

**Pull request merged**